### DISCUSSION TITLE
【代码块换行】某些主题中有些代码块在小屏幕下无法保持正常

### DISCUSSION THREAD

**Discussion by JIeJaitt** [tangly1024/NotionNext#2039] [Ideas]
**为什么提出这个新的特性改动**
在小屏幕移动端的时候一些主题，包括next，matery，hexo等在内。详情见https://github.com/tangly1024/NotionNext/discussions/887

**描述一下你推荐的解决方案**
（1）小屏幕下可以左右横移的那种
（2）或者代码块直接随着屏幕的缩小而整体缩小

**Comment by tangly1024** [comment]
代码显示行号的情况下是不允许横向滚动的；
若希望左右横移动，可以关闭此配置：
https://github.com/tangly1024/NotionNext/blob/720f2f7dba0362667139af7995c96755d00f09c4/blog.config.js#L64

**Comment by JIeJaitt** [comment]
> 代码显示行号的情况下是不允许横向滚动的； 若希望左右横移动，可以关闭此配置：
> 
> https://github.com/tangly1024/NotionNext/blob/720f2f7dba0362667139af7995c96755d00f09c4/blog.config.js#L64

若是希望能够做到既可以显示代码行号，又可以左右横向滚动请问可以嘛？希望可以列入未来的优化计划之一

**Comment by tangly1024** [comment]
> > 代码显示行号的情况下是不允许横向滚动的； 若希望左右横移动，可以关闭此配置：
> > https://github.com/tangly1024/NotionNext/blob/720f2f7dba0362667139af7995c96755d00f09c4/blog.config.js#L64
> 
> 若是希望能够做到既可以显示代码行号，又可以左右横向滚动请问可以嘛？希望可以列入未来的优化计划之一

有考虑过、并且尝试过这么做；
但这样处理，技术上有点困难，换行和显示行号的功能的在css样式中是冲突的，横向滚动时开启行号就会出现错乱。
有时间会再回头处理这个问题。
